test/integ/integ_test.go

```diff
@@ -0,0 +1,251 @@
+			"The verrazzano operator doesn't doesn't exist")
+	})
+	It(fmt.Sprintf("Pod prefixed by %s exists", verrazzanoOperator), func() {
+		Expect(K8sClient.DoesPodExist(verrazzanoOperator, verrazzanoSystem)).To(BeTrue(),
+			"The verrazzano operator pod doesn't exist")
+	})
+	It("VMI should exist ", func() {
+		Eventually(vmiExists, tenSeconds).Should(BeTrue())
+	})
+})
+
+var _ = Describe("logging namespace resources ", func() {
+	It("logging namespace should exist ", func() {
+		Eventually(loggingNamespaceExists, oneMinute).Should(BeTrue())
+	})
+	It("Filebeat daemonset should exist ", func() {
+		Eventually(filebeatDaemonsetExists, tenSeconds).Should(BeTrue())
+	})
+	It("Filebeat pod should exist ", func() {
+		Eventually(filebeatPodExists, oneMinute).Should(BeTrue())
+	})
+	It("Jounalbeat daemonset should exist ", func() {
+		Eventually(journalbeatDaemonsetExists, tenSeconds).Should(BeTrue())
+	})
+	It("Jounalbeat pod should exist ", func() {
+		Eventually(journalbeatPodExists, oneMinute).Should(BeTrue())
+	})
+})
+
+var _ = Describe("lonitoring namespace resources ", func() {
+	It("monitoring namespace should exist ", func() {
+		Eventually(monitoringNamespaceExists, oneMinute).Should(BeTrue())
+	})
+	It("Node exporter service should exist ", func() {
+		Eventually(nodeExporterServiceExists, tenSeconds).Should(BeTrue())
+	})
+	It("Node exporter pod should exist ", func() {
+		Eventually(nodeExporterPodExists, oneMinute).Should(BeTrue())
+	})
+	It("Node exporter daemonset should exist ", func() {
+		Eventually(nodeExporterDaemonExists, tenSeconds).Should(BeTrue())
+	})
+	It("Prom pusher deployment should exist ", func() {
+		Eventually(promPusherDeploymentExists, tenSeconds).Should(BeTrue())
+	})
+	It("Prom pusher pod should exist ", func() {
+		Eventually(promPusherPodExists, oneMinute).Should(BeTrue())
+	})
+})
+
+var _ = Describe("Testing generic app model/binding lifecycle", func() {
+	It("apply model should result in a vm in default namespace", func() {
+		_, stderr := util.RunCommand("kubectl apply -f testdata/gen-model.yaml")
+		Expect(stderr).To(Equal(""))
+		Eventually(genAppModelExists, fewSeconds).Should(BeTrue())
+	})
+	It("apply binding should result in a vb in default namespace", func() {
+		_, stderr := util.RunCommand("kubectl apply -f testdata/gen-binding.yaml")
+		Expect(stderr).To(Equal(""))
+		Eventually(genAppBindingExists, fewSeconds).Should(BeTrue())
+	})
+	It("genapp namespace should exist", func() {
+		Eventually(genAppNsExists, tenSeconds).Should(BeTrue())
+	})
+	It("genapp VMI should exist ", func() {
+		Eventually(genAppVmiExists, tenSeconds).Should(BeTrue())
+	})
+	It("genapp deployment should exist ", func() {
+		Eventually(genAppDeploymentExists, tenSeconds).Should(BeTrue())
+	})
+	It("genapp pod should exist ", func() {
+		Eventually(genAppPodExists, oneMinute).Should(BeTrue())
+	})
+	It("genapp prom pusher deployment should exist ", func() {
```

Comment:
should you check that no service exists for the generic component.  In your case there was no container ports specified in the generic component so no service is created

Reply:
will do